### DISCUSSION TITLE
Dockge and ram usage

### DISCUSSION THREAD

**Discussion by Nic-kh** [louislam/dockge#620] [General]
Hello friends!

Tell me, does the running Dockge , which has three content, take up so much memory?
![image](https://github.com/user-attachments/assets/21f00c93-7617-471d-8676-cc06215c1107)

the same portainer, a colossus with a bunch of dockers and takes up half as much space 🤷‍♂️
![image](https://github.com/user-attachments/assets/a58c847f-ad1f-4fe1-97bc-d5279a1aadc7)

I will be glad to receive any advice

*dockge:latest

**Comment by tismofied** [comment]
Same here as well.

**Reply by vytas911** [reply]
Same here

**Comment by kingp0dd** [comment]
I noticed the same issue just now.

**Comment by rubens-soto** [comment]
is this active problem? I was planning to start using dockge.

**Reply by clockwinder** [reply]
Did we ever find out? Did you test it?

**Reply by Nic-kh** [reply]
<img width="468" height="135" alt="image" src="https://github.com/user-attachments/assets/d5023a54-e1b7-47bd-89b7-167e284c0a15" />

It seems this isn't a problem and is a given.

I've come to terms with it.

**Reply by clockwinder** [reply]
I'm still hopeful it'll get looked into at some point.

**Reply by steveepreston** [reply]
Has memory issue gone?

**Comment by kingp0dd** [comment]
I use komodo now

**Reply by clockwinder** [reply]
And it's met all your needs?

**Comment by henfri** [comment]
Hello,

I also find that Ram-Usage of Dockge is very high.

Could you have a look, please?

Best regards,
Hendrik

**Comment by aryanrajseo** [comment]
switched from portainer thought it would take 50-60mb but i was wrong. other idle services have 40-100mb uses but dockge memory uses is way higher.  is it normal? dev should confirm. i might switch to komodo or dokploy or simply go back to portainer.

<img width="911" height="198" alt="image" src="https://github.com/user-attachments/assets/c49389fb-4a0b-4963-b555-5261ee23b5ac" />

**Comment by aryanrajseo** [comment]
here portainer uses only few mbs, this is from my different vm.

<img width="825" height="126" alt="image" src="https://github.com/user-attachments/assets/17bcf96b-58e0-44bd-96a4-967f4fe24c9f" />

**Comment by tribor** [comment]
I love dockerge for it clean look and really would consider using it... but the memory usage is a show stopper for my low-memory minipc. what is the reason for a simple gui using so much RAM?

**Reply by aryanrajseo** [reply]
i don't think dev is actively working. you can checkout dockhand. i have moved back to portainer.

**Reply by steveepreston** [reply]
@aryanrajseo ah! sad news

**Comment by Aerya** [comment]
Or you can have a look at my https://github.com/Aerya/Dockge-Enhanced
